app/native/app/(tabs)/couchers.tsx

```diff
@@ -0,0 +1,146 @@
+import { Image, StyleSheet, Button, Text } from "react-native";
+
+import ParallaxScrollView from "@/components/ParallaxScrollView";
+import { ThemedText } from "@/components/ThemedText";
+import { ThemedView } from "@/components/ThemedView";
+import { useRef, useState } from "react";
+import { useMutation } from "react-query";
+import { RpcError } from "grpc-web";
+import { service } from "service";
+import { StatusRes } from "proto/bugs_pb";
+import Alert from "components/Alert";
+import LoginForm from "features/auth/login/LoginForm";
+
+import { useAuthContext } from "features/auth/AuthProvider";
+
+import { useTranslation } from "i18n";
+import { WebView, WebViewNavigation } from "react-native-webview";
+import { AUTH, GLOBAL } from "i18n/namespaces";
+
+const WEB_BASE_URL = process.env.EXPO_PUBLIC_WEB_BASE_URL!;
+
+export default function CouchersScreen() {
+  const { t } = useTranslation([AUTH, GLOBAL]);
+
+  const [pressed, setPressed] = useState<number>(0);
+
+  const { authState, authActions } = useAuthContext();
+
+  let webview = useRef<WebView>(null);
+
+  const { data: resp, mutate: checkCoucherCount } = useMutation<
+    StatusRes.AsObject,
+    RpcError,
+    void
+  >(() => service.version.status());
+
+  const logIn = () => {
+    authActions.passwordLogin({
+      username: "aapeli_native",
+      password: "unsafepassword",
+      rememberDevice: true,
+    });
+  };
+
+  const logOut = () => {
+    authActions.logout();
+  };
+
+  const onPress = () => {
+    setPressed(pressed + 1);
+    // Alert.alert('Simple Button pressed')
+    checkCoucherCount();
+  };
+
+  const handleWebViewNavigationStateChange = (
+    newNavState: WebViewNavigation,
+  ) => {
+    const { url } = newNavState;
+    if (!url) return;
+    const v = webview.current;
+    if (!v) return;
+
+    if (!url.startsWith(WEB_BASE_URL)) {
+      v.stopLoading();
+      console.log("oooop");
+    }
+  };
+
+  return (
+    <ParallaxScrollView
+      headerBackgroundColor={{ light: "#A1CEDC", dark: "#1D3D47" }}
+      headerImage={
+        <Image
+          source={require("@/assets/images/couchers512.png")}
```

Comment:
Generally it's frown on to import like this, but I dunno is it a React native thing? Also usually for frontend you do the ES6 import rather than CommonJS imports, i.e. ` import something from "somewhere"`.